packages/Notification/Notification.modules.scss

```diff
@@ -28,8 +28,16 @@
   width: calc(100% - 2.5rem);
 }
 
+.withDismissIcon {
+  padding-right: 2rem;
+}
+
+.dismissContainer {
+  position: relative;
+}
+
 .dismiss {
   position: absolute;
-  top: 0.8rem;
-  right: 1rem;
+  top: -0.2rem;
+  right: 0rem;
```

Comment:
This could just be `right: 0`.

Reply:
Good catch, fixed in [c5747da](https://github.com/telus/tds-core/pull/917/commits/c5747da6fa5b2b08421bd981e4548f734b51383d)